src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -145,258 +133,194 @@ const AssigneeSelector = createReactClass({
-          />
-        </MenuItem>
-
-        {assignedTo && (
-          <MenuItem
-            className="clear-assignee"
-            disabled={!loading}
-            onSelect={this.clearAssignTo}
-          >
-            <MenuItemWrapper py={0}>
-              <IconContainer>
-                <ClearAssigneeIcon />
-              </IconContainer>
-              <Label>{t('Clear Assignee')}</Label>
-            </MenuItemWrapper>
-          </MenuItem>
-        )}
-
-        <li>
-          <ul>
-            {teams}
-            {hasTeamsAndMembers ? <Divider key="divider" /> : null}
-            {members}
-            {!hasTeamsOrMembers && (
-              <li className="not-found">
-                <span>{t('No matches found.')}</span>
-              </li>
-            )}
-          </ul>
-        </li>
-      </React.Fragment>
-    );
+    return [
+      {id: 'team-header', hideGroupLabel: true, items: teams},
+      {id: 'members-header', items: members},
+    ];
   },
 
   render() {
-    let {loading, assignedTo} = this.state;
-    let group = GroupStore.get(this.props.id);
-
-    let org = this.context.organization;
-    let access = new Set(org.access);
-
-    let assigneeListLoading = this.state.memberList === null || !group;
-
-    if (loading) {
-      return (
-        <div>
-          <div className="assignee-selector anchor-right">
-            <LoadingIndicator mini style={{marginRight: '10px'}} />
-          </div>
-        </div>
-      );
-    }
-
-    let className = classNames('assignee-selector anchor-right', {
-      unassigned: !assignedTo,
-    });
+    let {className} = this.props;
+    let {loading, assignedTo, memberList} = this.state;
+    let canInvite = ConfigStore.get('invitesEnabled');
 
     return (
       <div className={className}>
-        <DropdownLink
-          className="assignee-selector-toggle"
-          onOpen={this.onDropdownOpen}
-          onClose={this.onDropdownClose}
-          isOpen={this.state.isOpen}
-          alwaysRenderMenu={false}
-          title={
-            assignedTo ? (
-              <ActorAvatar actor={assignedTo} className="avatar" size={24} />
-            ) : (
-              <span className="icon-user" />
-            )
-          }
-        >
-          {assigneeListLoading ? (
-            <li>
-              <FlowLayout center className="list-loading-container">
-                <LoadingIndicator mini />
-              </FlowLayout>
-            </li>
-          ) : (
-            this.renderDropdownItems()
-          )}
-          {ConfigStore.get('invitesEnabled') &&
-            access.has('org:write') && (
-              <React.Fragment>
-                <Divider />
-                <MenuItem
-                  className="invite-member"
+        {loading && (
+          <LoadingIndicator mini style={{height: '24px', margin: 0, marginRight: 11}} />
+        )}
+        {!loading && (
+          <DropdownAutoComplete
+            maxHeight={400}
```

Comment:
can we change this to be an unambiguous unit? two reasons:

1. it's ambiguous
2. if we allow any unit then we can pass in `50vh` which would be pretty sweet (tho you will also need to add a static min/max height)

Reply:
I think we should support both and assume it's `px` when it is a number because sometimes you may need to do some calculations with the integer value and it's better to have the same interface for all components.